lux/extensions/admin/sidebar.py

```diff
@@ -212,10 +216,24 @@ def add_css(all):
         padding_bottom=px(10))
 
     media(max_width=collapse_width).css(
+        '.collapse',
+        display='block !important',
+        visibility='hidden',
+        transform='translate(-100px, 0)',
+        transition='all 0.15s cubic-bezier(0.2, 0.3, 0.25, 0.9) 0s',
+        height=px(0)).css(
+        '.collapse.in',
+        visibility='visible',
+        transform='translate(0, 0)',
+        transition='all 0.15s cubic-bezier(0.2, 0.3, 0.25, 0.9) 0s',
+        height=px(120)).css(
+        '.navbar-collapse.in',
+        overflow_y='hidden !important').css(
```

Comment:
This should be applied to the navbar right? what is it doing here?
Is the best location in https://github.com/quantmind/lux/blob/dev/lux/extensions/ui/css/navbar.py?